src/convert/to_text.cc

```diff
@@ -0,0 +1,740 @@
+//
+// Copyright 2020 WebAssembly Community Group participants
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "wasp/convert/to_text.h"
+
+#include <cassert>
```

Comment:
Kind of funny to see this old C header in an this otherwise ultra modern C++ code :)

Reply:
Well, to be fair "cassert" is the C++ equivalent of the C header "assert.h" :-)

Now that we use absl, we could switch to `ABSL_ASSERT` too: https://github.com/abseil/abseil-cpp/blob/master/absl/base/macros.h#L94